keras/engine/training.py

```diff
@@ -395,36 +391,85 @@ def standardize_weights(y, sample_weight=None, class_weight=None,
 
 
 def generator_queue(generator, max_q_size=10,
-                    wait_time=0.05, nb_worker=1):
-    '''Builds a threading queue out of a data generator.
+                    wait_time=0.05, nb_worker=2):
+    '''Builds a multiprocessing queue out of a data generator.
     Used in `fit_generator`, `evaluate_generator`, `predict_generator`.
+    Note that multiprocessing can't easily pass non picklable objects to
+    children processes. This notably means you can't pass the same lazy evaluated data array
+    to several children processes.
+
+    examples :
```

Comment:
If providing code examples in a docstring, it should use the MarkDown syntax for code snippets (as used elsewhere in docstring code snippets in the codebase) and the code should follow PEP8 syntax.

Alternatively, you could simply remove the code example, remove the mention of "lazy evaluated arrays", and simply state that the data should be picklable.